announcementsAPI.json

```diff
@@ -682,6 +769,20 @@
         },
         "additionalProperties": false
       },
+      "PutAdopterLikeAnnouncementRequest": {
```

Comment:
This JSON is either unnecessary or should be changed to contain only the "isLiked" field. If the second case is applicable, then "isLiked" should be removed from the query parameters of /announcements/{announcementId}/like endpoint and we should add this JSON as a request body.

Reply:
I forgot to delete this request, it should't be here